pkg/auth_test.go

```diff
@@ -105,20 +105,20 @@ func TestAuth_CreateContractSession(t *testing.T) {
 }
 
 func TestAuth_ContractByType(t *testing.T) {
+	sut := Auth{ValidContracts: Contracts}
 	t.Run("get contract by type", func(t *testing.T) {
-		sut := Auth{}
 		result, err := sut.ContractByType(ContractType("BehandelaarLogin"), Language("NL"), Version("v1"))
 
-		assert.Nil(t, err)
-		assert.NotNil(t, result)
+		if !assert.Nil(t, err) || !assert.NotNil(t, result) {
+			t.FailNow()
```

Comment:
you can just return here

Reply:
isn't this more explicit? Would you advice rewrite all FailNow's to returns?

Reply:
Hmm I just tend to use return, since the assert functions make the test fail anyways. Probably just a matter of style.